### PR TITLE
feat/jsMainKz/create login signUp pages

### PR DESCRIPTION
**Pull request by Tigran-Sarkisov**
create login and signUp pages

### CODE REVIEW

File: src/components/AuthForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            </h2>
+            <span className="text-center text-gray-100">
+              Введите адрес электронной почты, чтобы войти или создать учетную
+              запись
+            </span>
+            <Input placeholder={'yours@example.com'} />
+            <span className="text-xs text-center text-gray-50">
+              Регистрируясь, вы соглашаетесь с нашими условиями обслуживания и
+              политикой конфиденциальности.
+            </span>
```
Comment (ArmanSarkisov) [inline src/components/AuthForm/index.tsx:40]: where is the checkbox for 

Регистрируясь, вы соглашаетесь с нашими условиями обслуживания и
              политикой конфиденциальности.
              
              without checkbox it is not correct

File: src/components/AuthForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <form className="flex flex-col items-center space-y-6  border-8 border-gray-600 w-80 h-auto rounded-lg ">
+          <div className="px-8 flex flex-col items-center space-y-6">
+            <h2 className="text-center text-white text-3xl mt-3 mb-3">
+              JSMEIN.KZ
+            </h2>
+            <span className="text-center text-gray-100">
+              Введите адрес электронной почты, чтобы войти или создать учетную
+              запись
+            </span>
+            <Input placeholder={'yours@example.com'} />
```
Comment (ArmanSarkisov) [inline src/components/AuthForm/index.tsx]: `<Input placeholder="yours@example.com" />`

File: src/components/AuthForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <span className="text-xs text-center text-gray-50">
+              Регистрируясь, вы соглашаетесь с нашими условиями обслуживания и
+              политикой конфиденциальности.
+            </span>
+          </div>
+          <Button type={'button'} className={'w-full h-16 bg-blue-600 text-white'}>
```
Comment (ArmanSarkisov) [inline src/components/AuthForm/index.tsx]: `<Button type="button" className="w-full h-16 bg-blue-600 text-white">`

File: src/main.tsx [hunk 4]
```diff
@@ -26,6 +27,8 @@ ReactDOM.render(
       <ReactQueryDevtools initialIsOpen={false} />
       <Router>
         <Routes>
+          <Route path="/login" element={<Login />}/>
+          <Route path="/signUp" element={<Login />}/>
```
Comment (ArmanSarkisov) [inline src/main.tsx]: Each path needs to be a separate component.
You cant use Login for /login and /signup

/signUp => /signup

File: src/pages/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { AuthForm } from "@/components/AuthForm";
+import React from "react";
+
+
+export const Login =()=>{
+    return   <div>
+            <AuthForm />
+        </div>
+
```
Comment (ArmanSarkisov) [inline src/pages/Login.tsx]: use prettier

File: src/pages/SignUp.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { AuthForm } from "@/components/AuthForm";
+import React from "react";
+
+
+export const SignUp =()=>{
+    return   <div>
+            <AuthForm />
+        </div>
```
Comment (ArmanSarkisov) [inline src/pages/SignUp.tsx]: use Prettier

**Review by ArmanSarkisov — CHANGES_REQUESTED** [review]

File: src/components/AuthForm/index.tsx (hunk 2 above)
Comment (Tigran-Sarkisov) [reply src/components/AuthForm/index.tsx]: done

File: src/components/AuthForm/index.tsx (hunk 3 above)
Comment (Tigran-Sarkisov) [reply src/components/AuthForm/index.tsx]: done

File: src/main.tsx (hunk 4 above)
Comment (Tigran-Sarkisov) [reply src/main.tsx]: done

File: src/pages/Login.tsx (hunk 5 above)
Comment (Tigran-Sarkisov) [reply src/pages/Login.tsx]: done

File: src/pages/SignUp.tsx (hunk 6 above)
Comment (Tigran-Sarkisov) [reply src/pages/SignUp.tsx]: done

File: src/components/AuthForm/index.tsx (hunk 1 above)
Comment (Tigran-Sarkisov) [reply src/components/AuthForm/index.tsx:40]: done

File: src/pages/Login.tsx [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
-export const Login =()=>{
-    return   <div>
-            <AuthForm />
-        </div>
-    
-}
\ No newline at end of file
+export const Login = () => {
+  return (
+    <div>
```
Comment (devsteveaks) [inline src/pages/Login.tsx]: You can remove div

File: src/pages/SignUp.tsx [hunk 8]
```diff
@@ -1,9 +1,10 @@
...
-
-export const SignUp =()=>{
-    return   <div>
-            <AuthForm />
-        </div>
-}
\ No newline at end of file
+export const SignUp = () => {
+  return (
+    <div>
```
Comment (devsteveaks) [inline src/pages/SignUp.tsx]: Remove div

File: src/pages/Login.tsx (hunk 7 above)
Comment (Tigran-Sarkisov) [reply src/pages/Login.tsx]: done

File: src/pages/SignUp.tsx (hunk 8 above)
Comment (Tigran-Sarkisov) [reply src/pages/SignUp.tsx]: done

**Pull request merged**